### PR TITLE
Move contributors page to community, remove from home page.

### LINKED ISSUE #2312: Move contributors page to community, remove from home page.
Remove the current footer of pictures and contributors from the main page.
Relocate to the community page.
Reflect the OWNERS file with Current and Past headings
Remove roles, only show Name, GitHub Username, Company
Order by github username.

### PR DESCRIPTION
**Pull request by lenriquez**
Signed-off-by: Luis Enriquez <felipe096@gmail.com>

**What this PR does / why we need it**:
I am a new approver!!!

**Which issue(s) this PR fixes**
- Fixes #2312

**Special notes for your reviewer**:

image

**Release note**:
```
release-note
```

### CODE REVIEW

File: site/_layouts/community.html [hunk 1]
```diff
@@ -0,0 +1,34 @@
+<!DOCTYPE html>
+<html lang="en">
+{% include head.html %}
+
+<body id="{{ page.id }}">
+<div class="container-fluid site-outer-container">
```
Comment (ftovaro) [inline site/_layouts/community.html:6]: What about a little indentation for the `div` below the `body`? 😃

Comment (lenriquez) [reply site/_layouts/community.html:6]: thanks!

File: site/_includes/contributors.html [hunk 2]
```diff
@@ -1,6 +1,6 @@
 <div class="row">
   <div class="col">
-    <h2 class="mb-2">The {{ site.title }} Team</h2>    
+    <h2 class="mb-2">The {{ site.title }} Team</h2>
```
Comment (wwitzel3) [inline site/_includes/contributors.html:3]: Let's change this to read "Community Members" or something like that.

**Review by wwitzel3 — CHANGES_REQUESTED** [review]

File: site/_includes/contributors.html (hunk 2 above)
Comment (lenriquez) [reply site/_includes/contributors.html]: I changed to "Community Members"

**Review by GuessWhoSamFoo — APPROVED** [review]

**Review by wwitzel3 — APPROVED** [review]

**Pull request merged**